App.js

```diff
@@ -17,14 +18,29 @@ import AppNavigator from './navigation/AppNavigator';
 
 Amplitude.initialize(keys.AMPLITUDE);
 
-if (__DEV__) {
-  // override amplitude tracking
+const channel = Constants?.manifest?.releaseChannel;
+const emptyStorage = () =>
+  AsyncStorage.multiRemove(['@posts', '@media', '@groups', '@missions']);
+// override amplitude tracking
+if (channel === null || channel === undefined || channel === '') {
```

Comment:
i think we can just do
```suggestion
if (!channel) {
```
here